css/src/edit-block.scss

```diff
@@ -0,0 +1,91 @@
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
```

Comment:
This builds Tailwind in this plugin, instead of hotlinking it from `unpkg`. `npm run build` does tree-shaking, making the stylesheet much smaller.